src/style.css

```diff
@@ -2,2 +1,83 @@
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Roboto, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. You can add fallback fonts like this: `font-family: Roboto, Arial, sans-serif;`